api/rosetta/data_integration_test.go

```diff
@@ -194,10 +196,29 @@ func validateByHeader(t *testing.T, header flow.Header) validateBlockFunc {
 	return validateBlock(t, header.Height, header.ID().String())
 }
 
+func getUint64P(n uint64) *uint64 {
+	x := n
+	return &x
+}
```

Comment:
IMO, you can actually just use `return &n`, because `n` is copied on function entry, and the `&n` will trigger escape analysis and make sure it's put on the heap and won't be garbage-collected.